custom/enikshay/management/commands/drug_resistance_reconciliation.py

```diff
@@ -145,22 +148,31 @@ def reconcile_drug_resistance_cases(self, drug_resistance_cases, drug_id, occurr
                              "Picked first opened.")
 
     def _get_open_occurrence_case_ids_to_process(self):
-        from corehq.sql_db.util import get_db_aliases_for_partitioned_query
-        dbs = get_db_aliases_for_partitioned_query()
-        for db in dbs:
-            case_ids = (
-                CommCareCaseSQL.objects
-                .using(db)
-                .filter(domain=DOMAIN, type="occurrence", closed=False)
-                .values_list('case_id', flat=True)
-            )
-            num_case_ids = len(case_ids)
-            if self.log_progress:
-                print("processing %d docs from db %s" % (num_case_ids, db))
-            for i, case_id in enumerate(case_ids):
-                yield case_id
+        if self.person_case_ids:
```

Comment:
If person_case_ids is an empty list, should this show all or none?  Wondering if this line should be:
`if self.person_case_ids is not None:`
(or whether that's applicable at all)

Reply:
Before this command is queued its ensured that the `person_case_ids` is either None or a list of case ids. So i think it would never be an empty list.